low-level/framework/utils/utility.py

```diff
@@ -80,6 +83,54 @@ def get_os(self):
             else:
                 return None
 
+    def get_epoch_time(self, date, _time):
+        timestamp_format = '%m/%d/%Y %H:%M:%S'
+        timestamp = time.strptime('{} {}'.format(date, _time), timestamp_format)
+        return str(int(calendar.timegm(timestamp)))
+
+    def create_file(self, path):
+        """Create a file at specified path."""
+        dir = path[:path.rindex("/")]
+        if not os.path.exists(dir):
+            os.makedirs(dir)
+        if not os.path.exists(path):
+            file = open(path, "w+")
+            file.close()
+
+    def read_raw_filedata(self, file):
+        """Reads the data from the file and returns it as a string."""
+        if not os.path.isfile(file):
+            return str(None)
+
+        with open (file, "r") as datafile:
+            return str(datafile.read().replace('\n', ''))
+
+    def get_hostname(self):
```

Comment:
hope these functions added here are nothing but copy of duplicated but tested code historically. we would like to use that code obviously, to not to invent the wheel again

Reply:
Yes.. Currently these functions are the duplicated ones.. Later we can remove the functions from seperate files and directly can use it from here.

Reply:
Yes ok.